tests/e2e/README.md

```diff
@@ -17,20 +17,33 @@ Run `make test-e2e` to execute the tests.
 | `project create` | Y |
 | `project get` | Y |
 | `project delete` | Y |
+| `project list` | N |
+| `project set` | N |
 | `app create`  | Y  |
 | `app get`  | Y  |
 | `app set` | Y |
 | `app delete` | Y |
+| `app describe` | N |
 | `app list` | Y |
-| `catalog` | N |
-| `create --git` | N |
+| `catalog list` | Y |
+| `catalog search` | N |
+| `component get` | Y |
+| `component set` | Y |
+| `create --git` | Y |
 | `create --local` | Y |
```

Comment:
there is no test calling `odo create --local`

Reply:
Oh I think #437 replaces https://github.com/redhat-developer/odo/blob/master/tests/e2e/e2e_test.go#L192

I think I will undo that change and add a separate test for `create --git`?

Reply:
We should cover all uses.
We have 4 different commands that can create a component.

`odo create`
`odo create --local`
`odo create --binary`
`odo create --git`

The first two are doing almost the same, but we should have tests covering both of them.